portfolio/src/main/webapp/designs.html

```diff
@@ -0,0 +1,101 @@
+
+		<!-- Wrapper -->
+			<div id="wrapper">
+
+				<!-- Main -->
+                <section id="main">
+                    <header>
+                        <h3 class="subtitle">Design</h4>
+                    </header>
+
+                	<h4>This summer, I'm learning how to use Adobe Illustrator. I've been trying to make a design every day of the summer, either about my day or by following a tutorial. Here are the ones I've done so far!</h4>
+                    </br>
+                    </br>
+
+                    <div class="container">
+                        <img class="designpix" src="images/googlebike.png" width="500" height="375"/>
+                        <div class="overlay">
+                            <div class="text">May 18, 2020</div>
+                        </div>
+                    </div>
+
+                    <div class="container">
+                        <img class="designpix" src="images/plants.png" width="500" height="375"/>
+                        <div class="overlay">
+                            <div class="text">May 19, 2020</div>
+                        </div>
+                    </div>
+
+                    <div class="container">
+                        <img class="designpix" src="images/fern.png" width="500" height="375"/>
+                        <div class="overlay">
+                            <div class="text">May 20, 2020</div>
+                        </div>
+                    </div>
+
+                    <div class="container">
+                        <img class="designpix" src="images/family.png" width="500" height="375"/>
+                        <div class="overlay">
+                            <div class="text">May 21, 2020</div>
+                        </div>
+                    </div>
+
+                    <div class="container">
+                        <img class="designpix" src="images/planets.png" width="500" height="375"/>
+                        <div class="overlay">
+                            <div class="text">May 22, 2020</div>
+                        </div>
+                    </div>
+
+                    <div class="container">
+                        <img class="designpix" src="images/cookies.png" width="500" height="375"/>
+                        <div class="overlay">
+                            <div class="text">May 23, 2020</div>
+                        </div>
+                    </div>
+					
+
+				<!-- Footer -->
+					<footer id="footer">
+						<ul class="copyright">
+							<li>&copy; Mihira Patel</li>
+						</ul>
+					</footer>
+                </section>
+			</div>
+
+		<!-- Scripts -->
+			<script>
+				if ('addEventListener' in window) {
+					window.addEventListener('load', function() { document.body.className = document.body.className.replace(/\bis-preload\b/, ''); });
+					document.body.className += (navigator.userAgent.match(/(MSIE|rv:11\.0)/) ? ' is-ie' : '');
+				}
+			</script>
+
+	</body>
+    <!-- <div id="content">
```

Comment:
We generally don't commit commented out code. It tends to just accumulate if left unchecked :)

Worst case, you can always pull it out of the repo.